src/Banner.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+    <div class="announcement__banner" v-if="isVisible">
+        <li v-for="(announcement, index) in announcements">
```

Comment:
li inside div doesn't sound correct :see_no_evil: 

That being said, we could use the official NcNoteCard component?
https://nextcloud-vue-components.netlify.app/#/Components/NcNoteCard

Reply:
Wow, I never used this, I am not familiar with _all_ components, I think I found a use case for this 👍 
I'll check if this works

Reply:
I used the NcNoteCard now, I could use the subject of that component, but I instead opted for the previous way in order to avoid unnecessary linebreaks. The banner shouldn't take too much space, just hiding the menu is annoying enough.

I updated the description with new images, I let the old one here for reference. I'll resolve this conversation (I like the new look better!)

The old look:
![Bildschirmfoto vom 2024-03-20 11-25-17](https://github.com/nextcloud/announcementcenter/assets/104770531/e6ea8c88-4bec-497e-8c9b-d7ab801a53cb)